graph-db/src/utils/indexing/indexCmrCollection.js

```diff
@@ -0,0 +1,46 @@
+require('array-foreach-async')
+const { indexRelatedUrl } = require('./indexRelatedUrl')
+
+/**
+ * Given a collection from the CMR, index it into Gremlin
+ * @param {JSON} collection collection object from `items` array in cmr response
+ * @param {Gremlin Traversal Object} gremlin connection to gremlin server
+ * @returns
+ */
+exports.indexCmrCollection = async (collection, gremlin) => {
+  const { meta, umm } = collection
+  const { 'concept-id': conceptId } = meta
+  const { EntryTitle: entryTitle, DOI: doi, RelatedUrls: relatedUrls } = umm
+  const { DOI: doiDescription } = doi
+  let doiUrl = 'Not provided'
+  let datasetName = `${process.env.CMR_ROOT}/concepts/${conceptId}.html`
+
+  if (doiDescription) {
+    // Take the second element from the split method
+    const [, doiAddress] = doiUrl.split(':')
+    doiUrl = `http://doi.org/${doiAddress}`
```

Comment:
While http get redirected to https - shouldn't we use https instead?

Reply:
At first I though you should read the https://doi.org from the authority metadata field first and then if it doesn't exist use the hard coded value (shouldn't we put that into a config file or environment variable - although it isn't likely to change.) But then this is an international standard and there isn't another DOI body that issues these, so you don't need to do that.